src/mbgl/layout/symbol_feature.hpp

```diff
@@ -10,42 +10,19 @@ namespace mbgl {
 
 class SymbolFeature : public GeometryTileFeature {
 public:
-    SymbolFeature(const std::unique_ptr<GeometryTileFeature>& feature) :
-        id(feature->getID()),
-        type(feature->getType()),
-        geometry(feature->getGeometries()),
-        properties(feature->getProperties())
+    SymbolFeature(std::unique_ptr<GeometryTileFeature> feature_) :
+        feature(std::move(feature_)),
+        geometry(feature->getGeometries()) // we need a mutable copy of the geometry for mergeLines()
```

Comment:
☝️ @jfirebaugh note - mergeLines relies on mutating the SymbolFeatures' geometries, but getGeometries() returns a copy, so this bit of copying still needed to happen.